src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```diff
@@ -85,6 +86,19 @@ const ImageClassificationObservationTable = ({ uploadedFiles, handleRemoveFile }
     }
   }
 
+  const handleRemoveImage = (file) => {
+    databaseSwitchboardInstance
+      .deleteImage(projectId, file.id)
+      .then(() => {
+        const updatedImages = images.filter((f) => f.id !== file.id)
```

Comment:
this seems to at least partially fix some of the glitchiness . Instead of filtering from current state, creating a new reference

`setImages((prevImages) => prevImages.filter((f) => f.id !== file.id))`

Reply:
current bug:

1. create a new bpq record
2. upload images
3. delete image
4. many requests fire off and table glitches

if i refresh the page, and try and delete again, it works as it should

Reply:
yes that makes sense. ok getting closer. Last thing I noticed is if I have a few images, and delete them, the last remaining image sticks around until i refresh the page.